### PR TITLE
add nightly build section

### PR DESCRIPTION
**Pull request by jackjia-ibm**
Signed-off-by: Jack (T.) Jia <jack-tiefeng.jia@ibm.com>

Fixes issue #64.

### CODE REVIEW

**Comment by nannanli** [comment]
@jackjia-ibm Hi Jack, I made some edits to replace direct URLs with variables to keep consistent with other sections. Also tweaked some wording. PR is ready for your review: https://github.com/zowe/zowe.github.io/pull/72/files. Thanks for the updates!

File: index.md [hunk 1]
```diff
@@ -79,6 +79,17 @@ The easiest way to get started with Zowe is by downloading the convenience build
   {% endif %}
 {% endfor %}
 </details>
+<details>
+<summary>Nightly Builds</summary>
+<p>Please visit Zowe Artifactory nightly build folder to find the most recent build: <a class="white" href="https://zowe.jfrog.io/zowe/webapp/#/artifacts/browse/tree/General/libs-release-local/org/zowe/nightly">https://zowe.jfrog.io/zowe/webapp/#/artifacts/browse/tree/General/libs-release-local/org/zowe/nightly</a>.</p>
+<p>Please note:
+  <ul>
+    <li><i>Nightly builds might not be stable, it's for preview purpose.</i></li>
```
Comment (Joe-Winchester) [inline index.md]: Rephrase to `Nightly builds are made available to allow preview and early distribution of in-progress work items which may be functionally incomplete and unstable.  The coverage and successful execution of tests has not been guaranteed and the builds should be treated accordingly.`

File: index.md [hunk 2]
```diff
@@ -79,6 +79,17 @@ The easiest way to get started with Zowe is by downloading the convenience build
...
+    <li><i>You can find latest build status from <a class="white" href="https://openmainframeproject.slack.com/archives/CC9QW5NJE">#zowe-build Slack channel</a>.</i></li>
```
Comment (Joe-Winchester) [inline index.md]: Rephrase to "The latest build status is available from ...".   (i.e. remove the pronoun "You can find").

**Comment by jackjia-ibm** [comment]
@nannanli @Joe-Winchester thanks for the rephrase suggestions. All changes are in. Please check.

**Pull request merged**